### ISSUE TITLE
Project Name Clash

### ISSUE THREAD

**Issue by dlemmermann** [Col-E/FxSampler#2]
You might wanna reconsider the project's name as it clashes heavily with the FXSampler subproject of ControlsFX. I wanna start using your project and might blog / tweet about it. But this could get confusing for some people.

**Comment by Col-E** [comment]
Got any ideas?

- JFX Sampler
- FXDE (development environment)
- Fx Playground

**Comment by dlemmermann** [comment]
Should have CSS in the name CssPlaygroundFX maybe?

**Comment by dlemmermann** [comment]
It is for CSS development only, right?

**Comment by dlemmermann** [comment]
I also have to implement a quite extensive CSS stylesheet now for a customer project and was reminded again of a) your project here and b) a test app that is part of OpenJFX. It was written for Modena and the older stylesheet Caspian. I am exploring now if this could become its own open source project with support for any stylesheet.

<img width="1264" alt="Bildschirmfoto 2022-01-21 um 11 59 56" src="https://user-images.githubusercontent.com/9534301/150516072-4ab00aac-0099-49ac-8130-05186103241c.png">
<img width="1264" alt="Bildschirmfoto 2022-01-21 um 12 00 01" src="https://user-images.githubusercontent.com/9534301/150516079-c5674543-bb13-43f7-b4b7-d67ae19217ae.png">
<img width="1264" alt="Bildschirmfoto 2022-01-21 um 12 00 06" src="https://user-images.githubusercontent.com/9534301/150516091-82580725-516c-46a0-9fe6-9d5498a76c66.png">

**Comment by Col-E** [comment]
Yeah this is for JFX's css only. If the app you're showing off could be its own open source project / independent of the main branch of JFX that'd be a great utility. Would love to see a live feedback via file-watchers if that's not already supported since that's one of the main things I made this for.

**Comment by Col-E** [comment]
I'm more inclined to archive this project once ShowcaseFX is up and running 😉
